docs/source/development/style-guide/review-standards.rst

```diff
@@ -0,0 +1,62 @@
+.. _pull-request-review-standards:
+
+Pull Request and Review Standards
+---------------------------------
+
+Before any code is merged into the code base, it will need to be put up for pull request (PR) and reviewed. The pull request should be created following
+the :ref:`checklist for pull requests <checklist-for-contributors-and-reviewers-of-pull-requests>`.
+
+Before opening a pull request
+===========================
```

Comment:
```suggestion
=============================
```
I think this is now too short. It'd be nice if we had the doc-build PRs linked for easy viewing, or maybe just running the sphinx builder with warnings as an extra test so it could log this out?